pkgs/by-name/et/etesync-dav/package.nix

```
@@ -8,14 +8,14 @@
 }:
 python3Packages.buildPythonApplication rec {
   pname = "etesync-dav";
-  version = "0.34.0";
+  version = "0.35.0";
   format = "pyproject";
```

```suggestion
  pyproject = true;
```
would be cool if we could switch to the new setting